massgov/openmass
# DP-31561 add form names and hidden fields

https://github.com/massgov/mayflower/pull/1866

Added:
  - description: Add name attributes to forms.
    issue: DP-31561
  - description: Add hidden fields to userway signup form for Formstack submission. 
    issue: DP-31561


To Test:
- The Userway signup form should have two sets of hidden inputs, one set added by GTM after the page load (will be removed) another set prerendered in code (added by this PR)
- These forms have a name attribute: header search form, home page search form, feedback form
